AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/internal/core/handler/FragmentArgHandler.java

```diff
@@ -67,12 +86,18 @@ public void process(Element element, EFragmentHolder holder) {
 			argKey = fieldName;
 		}
 
-		TypeMirror actualType = codeModelHelper.getActualType(element, holder);
+		DeclaredType enclosingClassType;
+		if (param.getKind() == ElementKind.PARAMETER) {
```

Comment:
i only count two. this one here and the one in [ExtraHandler](https://github.com/excilys/androidannotations/pull/1457/files#diff-dd36efc89c87354095c6bea0c49233a4R120). not sure if it is worth a method in `InjectHelper`.